### PR TITLE
Update manage-office-365-message-encryption.md

### PR DESCRIPTION
**Pull request by msbemba**
Tested and made changes to the ps cmdlets as the encrypt button is disabled only after changing three parameters

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8327

File: microsoft-365/compliance/manage-office-365-message-encryption.md [hunk 1]
```diff
@@ -87,22 +87,30 @@ As an administrator, you can manage whether to display this button to end users.
   
 1. Use a work or school account that has global administrator permissions in your organization and start a Windows PowerShell session and connect to Exchange Online. For instructions, see [Connect to Exchange Online PowerShell](/powershell/exchange/connect-to-exchange-online-powershell).
 
-2. Run the Set-IRMConfiguration cmdlet with the -SimplifiedClientAccessEnabled parameter:
+2. Run the Set-IRMConfiguration cmdlet with the -SimplifiedClientAccessEnabled, -SimplifiedClientAccessEncryptionDisabled and -SimplifiedClientAccessDoNotForwardDisabled parameter:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/manage-office-365-message-encryption.md]:
```suggestion
2. Run the Set-IRMConfiguration cmdlet with the -SimplifiedClientAccessEnabled, -SimplifiedClientAccessEncryptionDisabled, and -SimplifiedClientAccessDoNotForwardDisabled parameter:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Pleas merge this PR. thanks

**Comment by yogkumgit** [comment]
@KCCross Gentle reminder. Thanks

**Comment by KCCross** [comment]
Thanks for testing this. I'm sending it to Samson for additional testing/review under other circumstances. Great find.

**Comment by yogkumgit** [comment]
Closing as the comment in #8327

**Pull request closed without merging**